DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -16,6 +17,9 @@
     using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
     using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current;
     using DigitalLearningSolutions.Web.ViewModels.LearningPortal.SelfAssessments;
+    using DocumentFormat.OpenXml.EMMA;

```

Comment:
What are these `DocumentFormat.OpenXml` libraries being used for?

Reply:
DocumentFormat.OpenXml must have been added while updating the controller method to be asynchronous.
Iv removed the using statements since they were not being used.
I've also moved the verification token check to an extension method.

Reply:
Thanks @OluwatobiAwe, looks much better. Before I merge this, can I get your opinion on whether we should be using this post pattern elsewhere/throughout the system to prevent repeat posting?

Reply:
**This would perfectly work anywhere**, However, I'll prefer we use an action attribute where an actual form is being posted, where a model error would simply be appended. I decided to use an extension in this case, because we didn't have a well-defined model in the Post request.